trieste/acquisition/function/greedy_batch.py

```diff
@@ -16,17 +16,21 @@
 """
```

Comment:
had quite a few comments here, I guess they were removed when you renamed the file - did you resolve all of them?

Reply:
unfortunately no - but the only ones pending were regarding adding CL, and how to deal with fantasize_method